go.mod

```diff
@@ -203,13 +203,10 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.7.0 // indirect
 )
 
-// gopacket/gopacket with:
-// - https://github.com/gopacket/gopacket/pull/101
-replace github.com/gopacket/gopacket => github.com/gopacket/gopacket v1.3.2-0.20241202175635-b43272ae1eb8
-
 // ebpf-go with some memory & cpu optimizations like
 // - https://github.com/cilium/ebpf/pull/1763
 // - https://github.com/cilium/ebpf/pull/1772
 // - https://github.com/cilium/ebpf/pull/1778
 // - https://github.com/cilium/ebpf/pull/1794
+// - https://github.com/cilium/ebpf/pull/1814
```

Comment:
nit: There is a new release that includes most of those PRs: https://github.com/cilium/ebpf/releases/tag/v0.19.0

Reply:
I left it out for this PR since it wasn't related to title!